src/fakeData/fakeHealthcareProfessionals.ts

```diff
@@ -86,3 +86,98 @@ export function generateSpokenLanguages({ count = 0, onlyEnglish = false } = {})
 
         )
 }
+
+export function generateAcceptedInsurance() {
+    return faker.helpers.enumValue(gqlTypes.Insurance)
+}
+
+export function generateFailingNames(validationToBeChecked:string, local: gqlTypes.Locale = gqlTypes.Locale.EnUs)
+    : gqlTypes.LocalizedNameInput {
+    let namesField: gqlTypes.LocalizedNameInput = {firstName: '', lastName: '', locale: gqlTypes.Locale.JaJp}
+    
+    switch (validationToBeChecked) {
+        case 'namesContainsInvalidAlphabet': {
```

Comment:
I used string for choosing what input should be created not sure if this is best practice.

Reply:
you should make an enum if you keep it this way, but I personally think just making a separate `generate....` methods for each is easier